Neos.Neos/Classes/Domain/Repository/SiteRepository.php

```diff
@@ -71,6 +71,23 @@ public function findFirstOnline()
         return $this->findOnline()->getFirst();
     }
 
+
+    /**
+     * @param string $nodeName
+     * @return Site|null
+     */
+    public function findOneByNodeName(string $nodeName): ?Site
```

Comment:
This is not directly related but it allows us to mock this call easier and the IDE likes this :)